bpa-demo/src/app/data-insights/pages/batch-performance-list/batch-performance-list.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="batch-list-header"><b>Batch Recipes and Equipments</b></div>
+<!-- Button to clear selection -->
+<!-- <button (click)="clearSelection()">Clear Selection</button> -->
+
+<!-- Batch changes-->
+<div class="batches-nav-bar">
+  <ng-container *ngFor="let tab of tabList">
+    <div class="batches-nav-bar-item" (click)="selectTab(tab.tabName)" [ngClass]="{selected: selectedTab === tab.tabName}">{{tab.tabName}}</div>
```

Comment:
avoid click on div. Against ARIA standards. Use button instead. Also I see this being used a lot. Create a reusable component for tabs if not there in flourish already